.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+
+before_install:
+  - composer self-update
+
+install:
+  - composer install --prefer-source --no-interaction --no-progress
```

Comment:
I included the `--prefer-source` because could be useful if some dependencies' development tools want be used, for example the Guzzle `tests/server.js`, so it's up to this repo owner, imo it's just a decision.

Composer cache: :+1: